packages/components/input/src/input.vue

```diff
@@ -129,6 +130,7 @@
         @blur="handleBlur"
         @change="handleChange"
         @keydown="handleKeydown"
+        @click.stop
```

Comment:
The focus and blur events of textarea are bound directly, not by proxy, so there seems to be no problem, and no modification is required.